hummingbot/gateway
# (feat) copy token list to conf folder

**Before submitting this PR, please make sure**:


**A description of the changes proposed in the pull request**:
The token-list used by the connectors changed the location to conf/lists. The user needs to copy and paste the files to get started with the gateway. 
This PR solves that issue by copying the files when doing the setup

**Tests performed by the developer**:
- Run gateway-setup.sh and check that the files are there.
- Build the docker image and start the container and check that the files where there too.


**Tips for QA testing**:
